### ISSUE TITLE
[Question] How to programmatically change the location of the cache folder?

### ISSUE THREAD

**Issue by wtlgo** [jeka-dev/jeka#236]
Hello!

I know that dependency management is not the main purpose of the `jeka-core` library, but out of all the dependency-management libraries I have tried to use so far, this one turned out to be the most convenient, so I decided to stick with it.

As I mentioned before, I use this library to resolve the dependency tree from maven central on runtime, and it works great. The only inconvenience I have with this library is being unable to figure out how to change the location where it saves all of the jars.

As far as I understood, it is possible by providing the `JEKA_CACHE_DIR` environmental variable, but it is not an option for me, because I am developing a module for a larger application, and I have no control over the environment where the application is going to run, that's why I need to be able to change that location at the runtime. Also, there's a possibility that other modules might depend on this library as well, and it would be wrong to interfere with them.

I tried to figure it out by inspecting the source code of `JkDependencyResolver`, but sadly it is a little bit too complex for me to understand, that's why I am asking for help.

Here's a snippet of how I use the library if it helps in any way.
```java
        final JkDependencySet deps = JkDependencySet.of()
                // and all dependencies
                .and("dependency last");
        final JkDependencyResolver resolver = JkDependencyResolver.of()
                .setRepos(JkRepo.ofMavenCentral().toSet());
        final List<Path> paths = resolver.resolve(deps).getFiles().getEntries();

        // proceeding with paths
```

Thank you.

**Comment by djeang** [comment]
Sorry for the late responese.

Is a system property for specifying this location will be ok for you ?

**Comment by djeang** [comment]
This has been released in 0.10.12.
You can reopen thes issue if this does not match your expectations.

**Comment by wtlgo** [comment]
Oh! I apologize for the late response as well. Yes, this absolutely works for me, thank you!
